pkgs/top-level/all-packages.nix

```diff
@@ -20827,7 +20827,12 @@ with pkgs;
 
   cpustat = callPackage ../os-specific/linux/cpustat { };
 
-  cockroachdb = callPackage ../servers/sql/cockroachdb { };
+  inherit (import ../servers/sql/cockroachdb pkgs)
+    cockroachdb_20_1
+    cockroachdb_20_2
+    cockroachdb_21_1
+  ;
+  cockroachdb = cockroachdb_20_1;
```

Comment:
If we keep 20.1 I think there should be a deprecation warning.

Reply:
20.1 is still maintained though until November. I don't know whether we should add deprecation warnings _before_ a version gets unmaintained (with the date of the removal), or _after_. I think _before_ is nicer.